src/Numerics/DGMethods/FVReconstructions.jl

```diff
@@ -0,0 +1,174 @@
+module FVReconstructions
+using KernelAbstractions.Extras: @unroll
+import StaticArrays: SUnitRange, SVector
+
+"""
+    AbstractReconstruction
+
+Supertype for FV reconstructions.
+
+Concrete types must provide implementions of
+    - `width(recon)`
+       returns the width of the reconstruction. Total number of points used in
+       reconstruction of top and bottom states is `2width(recon) + 1`
+       - (::AbstractReconstruction)(state_top, state_bottom, cell_states::SVector,
+                                    cell_weights)
+      compute the reconstruction
+
+(::AbstractReconstruction)(
+        state_top,
+        state_bottom,
+        cell_states,
+        cell_weights,
+    )
```

Comment:
It looks like some of this section could be wrapped in code blocks

Reply:
Good catch. Thanks!